Public/stylesheets/main.css

```diff
@@ -187,6 +187,7 @@ li.author:before { content: "\e901"; }
 li.history:before { content: "\e902" }
 li.activity:before { content: "\e903"; }
 li.products:before { content: "\e904"; }
+li.stars:before { content: "\e904"; }
```

Comment:
I'm not sure how the icon is being referenced - I've kept it referecing the product one.

Leaving the PR as draft so we can add in the icon and change this reference here - that's all it should take. Snapshots should be unaffected since they already use the correct class and don't display the actual icon in the screenshots.